### ISSUE TITLE
Sending keys / right click not working in WoW Classic TBC

### ISSUE THREAD

**Issue by dannyr-git** [julianperrott/FishingFun#41]
Bot works in every way on Wow classic, except it won't send the "4" key for cast fishing.

IT will move the mouse on bobber, but it won't right click.

**Comment by szabonight** [comment]
Yeah I have the same problem, Julian suggested (on youtube) that someone may know another way to code to get the bot to send another way?

**Comment by szabonight** [comment]
Will this help - I'm not a coder.
https://stackoverflow.com/questions/7732372/postmessage-to-current-active-process-doesnt-work-for-all-process

**Comment by szabonight** [comment]
I had to change my *.exe name but it started working after it was compiled run it as ADMIN

**Comment by virtyworld** [comment]
it's all was working for me, but today isn't. that's because i was changed in my settings "decimal symbol" from comma to dot(ru region) for example check this https://www.officetooltips.com/office_2016/tips/how_to_change_decimal_symbol_and_digit_grouping_symbol_in_windows_10.html
